manager/install_cortex.sh

```diff
@@ -16,6 +16,10 @@
 
 set -e
 
+function ping_install() {
+  curl --output /dev/null -k -X POST -H "Content-Type: application/json" $CORTEX_TELEMETRY_URL/events -d '{"timestamp": "'$(date -u +"%Y-%m-%dT%H:%M:%SZ")'", "event": "setup.install_start", "version": "'$CORTEX_VERSION_STABLE'"}'
```

Comment:
Will this log if there is an error? Maybe adding `>/dev/null 2>&1` at the end would be safer than `--output /dev/null`